### ISSUE TITLE
KDE Desktop is missing online-account setting

### ISSUE THREAD

**Issue by sandersr** [clearlinux/distribution#1606]
Please include this in your KDE build

https://userbase.kde.org/System_Settings/Online_Accounts

**Comment by ahkok** [comment]
@anselmolsm maybe can look into this?

**Comment by anselmolsm** [comment]
Does that provide anything beyond accounts configuration for telepathy (i.e. Instant messaging accounts)? I ask because we have recently removed a few telepathy-related packages due to the dependency on python 2...

**Comment by sandersr** [comment]
Yeah it does. My main use is Google drive integration. Without the ability
to add Google account, I cannot access Google drive via kio.

**Comment by anselmolsm** [comment]
@sandersr thanks for the update, sounds reasonable.

**Comment by anselmolsm** [comment]
Packaged accounts-qml-module, kaccounts-integration, kaccounts-providers, kio-gdrive, libaccounts-glib, libaccounts-qt, signon-kwallet-extension, signon-plugin-oauth2, signon-ui, and signond. Some of them still need tweaks for stateless configs.

For the gdrive case specifically, due to https://gitlab.com/accounts-sso/signon-ui/issues/2 , https://bugs.kde.org/show_bug.cgi?id=414219 login is not working with error message "This browser or app may not be secure."

**Comment by sandersr** [comment]
Thank you very much for doing it! Much appreciated.

**Comment by anselmolsm** [comment]
@sandersr you're welcome. Please note that the packages are still not ready to go, I'll post updates here.

**Comment by sandersr** [comment]
Hi,

Just wanted to check how's this progressing. Let me know if there's anything I can help with.

**Comment by anselmolsm** [comment]
Still need work to make sure all packages work according to Clear's stateless design.
